common/views/components/Body/Body.tsx

```diff
@@ -57,6 +57,13 @@ import { convertItemToCardProps } from '@weco/common/model/card';
 import ConditionalWrapper from '../ConditionalWrapper/ConditionalWrapper';
 import VisitUsStaticContent from './VisitUsStaticContent';
 import CollectionsStaticContent from './CollectionsStaticContent';
+import styled from 'styled-components';
+
+export const BodyPart = styled.div`
```

Comment:
Are these even needed? display: block seems redundant, z-index might be a hangover from the right column stuff

Reply:
yeah, I did wonder. Couldn't be sure though. I suppose we could get rid altogether and be primed to add it back in if there was something we'd overlooked?